scripts/read_ledger

```diff
@@ -89,6 +88,7 @@ def get_ledger_dir(node_name, client_name, network):
 
 def get_storage(type_, ledger_data_dir):
     config = getConfig()
+    postfix = config.transactions_file_base_postfix
```

Comment:
For now it may be better to have a hard-coded constant here, so that we can provide the script as is for debug purposes to test pools without a need to re-install IndyNode.
Add a TODO comment to get rid of the constant.

Reply:
ok